### DISCUSSION TITLE
Azure Visual Studio Subscriptions card number not matched

### DISCUSSION THREAD

**Discussion by MdNadeemSarwar** [microsoft/SATechnicalOnboarding#128] [Questions]
when we are working on the Visual Studio Subscriptions page and try to activate the Azure account so in the identity verification I get a Error like **Check the info you entered. It doesn't match the info for this card.** How can I solve ?

**Comment by AnthonyByansi** [comment]
Hopefully this has been addressed here https://github.com/microsoft/SATechnicalOnboarding/issues/127#issuecomment-2071256162
